code/ai/ai_flags.h

```diff
@@ -110,6 +110,7 @@ namespace AI {
         Smart_shield_management,
         Smart_subsystem_targeting_for_turrets,
         Strict_turred_tagged_only_targeting,
+		Support_dont_add_primaries,
```

Comment:
A small comment explaining what this flag would be useful here. Maybe with the attached mantis issue number if someone wants to look up the reason for having this feature.